src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/NotificationsController.java

```diff
@@ -178,9 +178,18 @@
         Map<String, Object> caseData = caseDetails.getData();
 
         if (caseDataService.isApplicantSolicitorAgreeToReceiveEmails(caseDetails)) {
-            log.info("Sending email notification to Solicitor for 'Consent Order Made'");
-            notificationService.sendConsentOrderMadeConfirmationEmail(caseDetails);
+            if(caseDataService.isConsentedApplication(caseDetails)) {
```

Comment:
make sure you've updated tests to mock the response of this new call out, you can also combine with the if statement above i.e. if something && something, rather than 2 separate ifs